index.js

```diff
@@ -100,11 +101,18 @@ function addExceptionsToUnprotected (exceptions) {
 }
 
 const listData = JSON.parse(fs.readFileSync(`${LISTS_DIR}/${unprotectedListName}`))
-defaultConfig.unprotectedTemporary = listData.exceptions
-
-addExceptionsToUnprotected(defaultConfig.unprotectedTemporary)
+addExceptionsToUnprotected(listData.exceptions)
 addExceptionsToUnprotected(defaultConfig.features.contentBlocking.exceptions)
 
+const excludedFeaturesFromUnprotectedTempExceptions = ['adClickAttribution', 'appTrackerProtection', 'autofill', 'duckPlayer', 'incontextSignup',
```

Comment:
This is a nit but could we break this list out to one entry per line for readability?